### PR TITLE
Vision doc thanks page

### LINKED ISSUE #117: create a vision doc thanks page
I want to include (a) all people who commented on issues, (b) all people who opened PRs, and (c) all people who joined an AVD writing session!

### PR DESCRIPTION
**Pull request by Stupremee**
Resolves #117

### CODE REVIEW

**Comment by nikomatsakis** [comment]
This is awesome! Thanks!

File: get_contributors.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    exit 1
+  fi
+}
+
+check_bin curl
+check_bin jq
+
+# Get a list of users that participated in issues.
+function issue_contributors() {
+  local numbers=$(curl -s -u $user:$token -H "Accept: application/vnd.github.v3+json" 'https://api.github.com/repos/rust-lang/wg-async-foundations/issues?state=all&labels=status-quo-story-ideas' | jq '.[].number')
```
Comment (nikomatsakis) [inline get_contributors.sh:39]: Hmm, I was reading over this and I got to thinking -- this doesn't include commenting on PRs, does it?

Comment (nikomatsakis) [reply get_contributors.sh:39]: Do we want to include that?

Comment (nikomatsakis) [reply get_contributors.sh:39]: Also, if so, should we just include all PRs or do we want to label those that are contributing stories...

**Review by nikomatsakis — COMMENTED** [review]
I know I merged this, but then I had a question :)

File: get_contributors.sh (hunk 1 above)
Comment (Stupremee) [reply get_contributors.sh:39]: It does include all pull requests with the `status-quo-story` label. GitHubs REST API v3 treats every pull request as an issue.

Comment (nikomatsakis) [reply get_contributors.sh:39]: OK, so if we just go and add those labels, we should be all set...


...we should extend it with shiny-future labels too I guess

Comment (Stupremee) [reply get_contributors.sh:39]: Yes that makes sense. Should I open a new PR that fixes it?

Comment (nikomatsakis) [reply get_contributors.sh:39]: Yep, thanks.

Comment (Stupremee) [reply get_contributors.sh:39]: @nikomatsakis I just noticed that there is no `shiny-story` label yet. You'ld have to create one

Comment (nikomatsakis) [reply get_contributors.sh:39]: Done

**Pull request merged**